### PR TITLE
hack: workaround for #492 (don't merge)

### PR DESCRIPTION
**Pull request by pwbs**
Well, this makes the problem described in https://github.com/ocsigen/eliom/issues/492 go away.
I'm not sure at all that it's the right approach, but it's a good lead to finding the real problem.

### CODE REVIEW

File: src/lib/eliom_client.client.ml [hunk 1]
```diff
@@ -933,7 +933,7 @@ let change_page (type m)
            update_session_info l l';
            run_callbacks (flush_onchangepage ());
            let%lwt () = f get_params post_params in
-           change_url_string_protected ~replace uri;
+(*           change_url_string_protected ~replace uri; <---- shouldn't be done, perhaps???? *)
```
Comment (vasilisp) [inline src/lib/eliom_client.client.ml]: We have just called a service, and we do need to change the URL accordingly.

**Comment by vasilisp** [comment]
Thanks for investigating. We are getting closer. We cannot accept either of these patches, though. We need to respect `?absolute` and produce proper URLs.

**Comment by pwbs** [comment]
> Thanks for investigating. We are getting closer. We cannot accept either of these patches, though. We need to respect `?absolute` and produce proper URLs.

I agree. This patch, as it is, is not meant to be pulled, it's only elements from the investigation :)

File: src/lib/eliom_client.client.ml (hunk 1 above)
Comment (pwbs) [inline src/lib/eliom_client.client.ml]: This (i.e., removing that line) breaks the login process! :(

**Pull request closed without merging**